static/src/js/components/KeyboardShortcutsModal/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Modal, Container,
+  Row, Col, Button
+} from 'react-bootstrap'
+import config from './config.json'
+import './KeyboardShortcutsModal.scss'
+
+export class KeyboardShortcutsModal extends Component {
+  constructor(props) {
+    super(props)
+    this.keyboardShortcuts = {
+      toggleModal: '?',
+      escapeModal: 'Escape'
+    }
+
+    this.onWindowKeyDown = this.onWindowKeyDown.bind(this)
+  }
+
+  componentDidMount() {
+    window.addEventListener('keydown', this.onWindowKeyDown)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('keydown', this.onWindowKeyDown)
+  }
+
+  onWindowKeyDown(e) {
+    const { key } = e
+    const { keyboardShortcuts } = this
+    const { isOpen, onToggleKeyboardShortcutsModal } = this.props
+    // Toggle the modal when question key pressed
+    if (key === keyboardShortcuts.toggleModal) {
+      onToggleKeyboardShortcutsModal(!isOpen)
+      e.preventDefault()
+      e.stopPropagation()
+    } else if (key === keyboardShortcuts.escapeModal) {
+      if (isOpen) onToggleKeyboardShortcutsModal(false)
+      e.preventDefault()
+      e.stopPropagation()
+    }
+  }
+
+
+  render() {
+    const { isOpen, onToggleKeyboardShortcutsModal } = this.props
+    const keyboardShortcutsList = config
+    return (
+      <Modal
```

Comment:
We have a component that we use for modal across the application called `EDSCModal`, please use that component to ensure consistent behavior and styling.